server/routes.go

```diff
@@ -1037,6 +1037,7 @@ func (s *Server) GenerateRoutes() http.Handler {
 	r.POST("/api/blobs/:digest", s.CreateBlobHandler)
 	r.HEAD("/api/blobs/:digest", s.HeadBlobHandler)
 	r.GET("/api/ps", s.ProcessHandler)
+	r.GET("/api/config", s.ConfigHandler)
```

Comment:
Let's avoid adding an API endpoint if we can. This is more of an optimization and so it should be under the hood.